local.Dockerfile

```diff
@@ -0,0 +1,47 @@
+# Builds images used by the bridge using locally built binaries.
+#
+# In particular, it can be used to build Substrate nodes and bridge relayers. The binary that gets
+# built can be specified with the `PROJECT` build-arg. For example, to build the `substrate-relay`
+# you would do the following:
+#
+# `docker build ./target -f local.Dockerfile -t local/substrate-relay --build-arg=PROJECT=substrate-relay`
+#
+# See the `deployments/README.md` for all the available `PROJECT` values.
+
+FROM docker.io/library/ubuntu:20.04 as runtime
```

Comment:
Nit: This needs to be the same as the local OS. Maybe it would be worth having a comment about this.

Reply:
Moved compatibility comment here. Out of curiosity - `substrate-relay` that is built on 23.04 doesn't work on 20.04?

Reply:
Doesn't seem to work. At least I'm getting the following error:

```
./substrate-relay: error while loading shared libraries: libssl.so.3: cannot open shared object file: No such file or directory
The command '/bin/sh -c ./${PROJECT} --version' returned a non-zero code: 127
```